modules/images/webp-uploads/image-edit.php

```diff
@@ -272,7 +273,7 @@ function webp_uploads_update_attachment_metadata( $data, $attachment_id ) {
  * @return array The updated metadata for the attachment.
  */
 function webp_uploads_backup_sources( $attachment_id, $data ) {
-	$target = isset( $_REQUEST['target'] ) ? sanitize_key( $_REQUEST['target'] ) : 'all';
+	$target = isset( $_REQUEST['target'] ) ? sanitize_key( $_REQUEST['target'] ) : 'all'; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
This needs an in-line comment, and preferably a reference to where nonce verification is being handled.